content/guides/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: IPFS Guides
+headless: true
+opened: true
+---
+
+# [Install IPFS](https://docs.ipfs.io/guides/install/)
+
+Install page describes the variety of ways you can have IPFS on the machine.
+The items are described below
+
+- Installing from a Prebuilt Package
+- Installing from ipfs-update
+- Building from source
+- Upgrading IPFS
+- Troubleshooting
+
+You should be using command line for these instructions.
+
+# [Transferring a File](https://github.com/ipfs/go-ipfs/blob/master/docs/file-transfer.md)
+
+# [Repliacting Large Datasets](https://github.com/ipfs/archives/tree/master/tutorials/replicating-large-datasets)
+
+# [Replicating Large Datasets](https://github.com/ipfs/archives/tree/master/tutorials/replicating-large-datasets)
+
+# [Addressing in Browsers](https://github.com/ipfs/in-web-browsers/blob/master/ADDRESSING.md)
+
```

Comment:
_Please add as in @ericronne comment:_
An overview of methods for browser-based content access